### ISSUE TITLE
Compile error when using gl_legacy

### ISSUE THREAD

**Issue by DuendeInexistente** [sm64pc/sm64ex#383]
**Describe the bug**
Compiling 55a0426cddc40157a2947d6c498aa731e06cc99e using render_api=gl_legacy errors out during build.

**To Reproduce**
1. Build using ```make RENDER_API=GL_LEGACY TEXTURE_FIX=1 BETTERCAMERA=1 EXTERNAL_DATA=1 DISCORDRPC=1 -j4```
2. Make errors out with
```
src/pc/gfx/gfx_opengl_legacy.c:592:13: error: redefinition of ‘gfx_opengl_shutdown’
 static void gfx_opengl_shutdown(void) {
             ^~~~~~~~~~~~~~~~~~~
src/pc/gfx/gfx_opengl_legacy.c:589:13: note: previous definition of ‘gfx_opengl_shutdown’ was here
 static void gfx_opengl_shutdown(void) {
             ^~~~~~~~~~~~~~~~~~~
src/pc/gfx/gfx_opengl_legacy.c:604:5: warning: initialization from incompatible pointer type [-Wincompatible-pointer-types]
     gfx_opengl_upload_texture,
     ^~~~~~~~~~~~~~~~~~~~~~~~~
src/pc/gfx/gfx_opengl_legacy.c:604:5: note: (near initialization for ‘gfx_opengl_api.upload_texture’)
Makefile:962: recipe for target 'build/us_pc/src/pc/gfx/gfx_opengl_legacy.o' failed
make: *** [build/us_pc/src/pc/gfx/gfx_opengl_legacy.o] Error 1

```

**Desktop (please complete the following information):**
 - OS: Ubuntu artful

**Comment by fgsfdsfgs** [comment]
Should be fixed in 8543ee8. Not sure how the fuck that happened.

**Comment by DuendeInexistente** [comment]
Alright, the fix worked.
